devdocs/process.md

```diff
@@ -68,8 +66,9 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
    to your branch
         
        ```
-       git pull upstream 
-       git merge master
+       git fetch upstream 
+       git checkout master
+       git merge upstream/master
```

Comment:
@kanghj is right. The `pull` will auto merge `upstream/master` to local `master`. But we still have to merge local `master` into the issue branch.